nvflare/fuel/f3/README.rst

```diff
@@ -0,0 +1,33 @@
+FLARE Federation Framework (F3)
```

Comment:
maybe we can add this file later after we re-factor

Reply:
Yeah. Removed.